scripts/src/admin/FormList/FormList.scss

```diff
@@ -100,3 +100,22 @@
     border: 1px solid black;
     float: left;
 }
+
+.help{
+    text-decoration: none !important;
+    color:black !important;
+    float: right;
+}
+
+@media (min-width: 768px ) {
+    .row {
+        position: relative;
+    }
+  
+    .bottom-align-text {
+      position: absolute;
+      bottom: 0;
+      right: 15px;
+      top:600px;
```

Comment:
This line of code is adding a large blank space at the bottom of the form:

![image](https://user-images.githubusercontent.com/1689183/62426627-2b5a4380-b69c-11e9-8196-677578f1664a.png)